### ISSUE TITLE
After upgrade to 3.2.14 Rabbit Transit images in grid is not showing

### ISSUE THREAD

**Issue by oefterdal** [akeneo/pim-community-dev#10942]
After upgrade we just get the default Akeneo thumbnail in product grid and inside the product page, if we hit download the image gets downloaded successfully and it's correct. 

**Here is the system info:** 

[Edition]
CE

[Version]
3.2.14

[Environment]
prod

[Install time]
2019-08-16T08:58:37+0000

[Server version]
nginx/1.14.0

[Number of channels]
2

[Number of locales]
1

[Number of products]
4762

[Number of product models]
0

[Number of variant products]
0

[Number of families]
2

[Number of users]
10

[Number of categories]
285

[Number of category trees]
3

[Max number of categories in one category]
33

[Max number of category levels]
4

[Number of product values]
145880

[Average number of product values by product]
31

[Average number of potential product values by family]
56

[Max number of potential product values by family]
57

[Number of attributes]
49

[Number of scopable attributes]
1

[Number of localizable attributes]
0

[Number of localizable and scopable attributes]
7

[Number of attributes useable as grid filter]
36

[Average percentage of scopable attributes per family (%)]
3

[Average percentage of localizable attributes per family (%)]
0

[Average percentage of localizable and scopable attributes per family (%)]
15

[Average number of attributes per family]
48

[OS version]
Linux pim-dev 4.15.0-1052-aws #54-Ubuntu SMP Tue Oct 1 15:43:26 UTC 2019 x86_64

[PHP version]
7.2.23-1+ubuntu18.04.1+deb.sury.org+1

[MySQL version]
5.7.27-0ubuntu0.18.04.1

[PHP extensions]
Core
date
libxml
openssl
pcre
zlib
filter
hash
Reflection
SPL
sodium
session
standard
cgi-fcgi
mysqlnd
PDO
xml
apcu
bcmath
calendar
ctype
curl
dom
mbstring
fileinfo
ftp
gd
gettext
iconv
imagick
intl
json
exif
mysqli
pdo_mysql
apc
posix
readline
shmop
SimpleXML
soap
sockets
sysvmsg
sysvsem
sysvshm
tokenizer
wddx
xmlreader
xmlwriter
xsl
zip
Phar
Zend OPcache

[Registered bundles]
Akeneo\Channel\Bundle\AkeneoChannelBundle
Akeneo\Pim\Enrichment\Bundle\AkeneoPimEnrichmentBundle
Akeneo\Pim\Structure\Bundle\AkeneoPimStructureBundle
Akeneo\Platform\Bundle\AnalyticsBundle\PimAnalyticsBundle
Akeneo\Platform\Bundle\CatalogVolumeMonitoringBundle\PimCatalogVolumeMonitoringBundle
Akeneo\Platform\Bundle\DashboardBundle\PimDashboardBundle
Akeneo\Platform\Bundle\ImportExportBundle\PimImportExportBundle
Akeneo\Platform\Bundle\InstallerBundle\PimInstallerBundle
Akeneo\Platform\Bundle\NotificationBundle\PimNotificationBundle
Akeneo\Platform\Bundle\UIBundle\PimUIBundle
Akeneo\Tool\Bundle\ApiBundle\PimApiBundle
Akeneo\Tool\Bundle\BatchBundle\AkeneoBatchBundle
Akeneo\Tool\Bundle\BatchQueueBundle\AkeneoBatchQueueBundle
Akeneo\Tool\Bundle\BufferBundle\AkeneoBufferBundle
Akeneo\Tool\Bundle\ClassificationBundle\AkeneoClassificationBundle
Akeneo\Tool\Bundle\ConnectorBundle\PimConnectorBundle
Akeneo\Tool\Bundle\ElasticsearchBundle\AkeneoElasticsearchBundle
Akeneo\Tool\Bundle\FileStorageBundle\AkeneoFileStorageBundle
Akeneo\Tool\Bundle\MeasureBundle\AkeneoMeasureBundle
Akeneo\Tool\Bundle\StorageUtilsBundle\AkeneoStorageUtilsBundle
Akeneo\Tool\Bundle\VersioningBundle\AkeneoVersioningBundle
Akeneo\UserManagement\Bundle\PimUserBundle
Doctrine\Bundle\DoctrineBundle\DoctrineBundle
Doctrine\Bundle\DoctrineCacheBundle\DoctrineCacheBundle
Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle
Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle
FOS\JsRoutingBundle\FOSJsRoutingBundle
FOS\OAuthServerBundle\FOSOAuthServerBundle
FOS\RestBundle\FOSRestBundle
Liip\ImagineBundle\LiipImagineBundle
Oneup\FlysystemBundle\OneupFlysystemBundle
Oro\Bundle\ConfigBundle\OroConfigBundle
Oro\Bundle\DataGridBundle\OroDataGridBundle
Oro\Bundle\FilterBundle\OroFilterBundle
Oro\Bundle\PimDataGridBundle\PimDataGridBundle
Oro\Bundle\PimFilterBundle\PimFilterBundle
Oro\Bundle\SecurityBundle\OroSecurityBundle
Oro\Bundle\TranslationBundle\OroTranslationBundle
Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle
Symfony\Bundle\FrameworkBundle\FrameworkBundle
Symfony\Bundle\MonologBundle\MonologBundle
Symfony\Bundle\SecurityBundle\SecurityBundle
Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle
Symfony\Bundle\TwigBundle\TwigBundle

**Comment by jmleroux** [comment]
Hello,
I know it's a common question but did you refresh you cache?
The easiest is to run these commands in console:
```
rm -rf app/cache/*
```

Also, you can try to reload/restart fpm because sometimes APC cache is not refreshed.
And finally, you can also refresh your front assets, by using the `bin/pim-front.sh` script.

**Comment by oefterdal** [comment]
Yes it’s cleared, I have even rebooted the server. I will try that script.

**Comment by srjwebster** [comment]
I have the same issue here on our latest Akeneo install. Will try clear the cache and reload the front assets.

The front end URL for an image attribute in the new pim is:
/media/show/1%252F5%252Fd%252F7%252F15d7b881b2467a4205f8518880f08da66f22dd7f_2794.jpg/thumbnail_small
whereas the download link is:
/media/download/1%252F5%252Fd%252F7%252F15d7b881b2467a4205f8518880f08da66f22dd7f_2794.jpg

When accessing the media/show/* version of the url I get a 500 error, whereas the download link downloads the expected image.

**Comment by oefterdal** [comment]
I get the same result @srjwebster. Haven't found a solution yet

**Comment by srjwebster** [comment]
On a working version when visiting that address you're forwarded to the location of the file on the server (in a series of directories with single character varchar names), on the 3.13 release this address can be visited manually, but the redirect doesn't work.

**Comment by oefterdal** [comment]
@jmleroux do you have any other tips?

**Comment by jmleroux** [comment]
Yep :)

I found the issue here : https://github.com/akeneo/pim-community-dev/blob/3.2/src/Akeneo/Pim/Enrichment/Bundle/Controller/Ui/FileController.php#L100

A bad return. Dunno if it's due to a bad pullup or a bad test, but i will fix it.

Stay tuned.

**Comment by jmleroux** [comment]
Fixed in #10948 

CI is running

**Comment by oefterdal** [comment]
Awesome support!

**Comment by srjwebster** [comment]
Good stuff guys, beat me too it. I was close, hard to follow on server without IDE completion. Just patched mine directly on server, will upgrade version later.

Found the file in /src/Akeneo/Pim/Enrichment/Bundle/Controller/Ui/FileController.php

**Comment by jmleroux** [comment]
> Found the file in /src/Akeneo/Pim/Enrichment/Bundle/Controller/Ui/FileController.php

You were very close. I have the luck to have all versions up at all time, so that's why i was the first. But it is a short win! :stuck_out_tongue_winking_eye:

**Comment by jmleroux** [comment]
Green CI.
I will deploy it on a test server before tagging. You can also test the branch on your side if you want.

**Comment by oefterdal** [comment]
@jmleroux  Works, closing this now. Thank you again.
